### ISSUE TITLE
pkg_resources.DistributionNotFound: The 'pycurl' distribution was not found and is required by DeeFuzzer

### ISSUE THREAD

**Issue by robnoordt** [yomguy/DeeFuzzer#91]
Hi, installed the deefuzzer on raspberry 3 debian stretch. The following erroor when starting the program. Pleace can some one give a stap bi stap solution? I am not realy good whit python. Hope to here soon!
pkg_resources.DistributionNotFound: The 'pycurl' distribution was not found and is required by DeeFuzzer
#

**Comment by robnoordt** [comment]
Here the full output:

root@raspberry3:~# deefuzzer /etc/deefuzzer/setup.pi
Traceback (most recent call last):
  File "/usr/local/bin/deefuzzer", line 4, in <module>
    __import__('pkg_resources').run_script('DeeFuzzer==0.7.2', 'deefuzzer')
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 3251, in <module>
    @_call_aside
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 3235, in _call_aside
    f(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 3264, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'pycurl' distribution was not found and is required by DeeFuzzer
root@raspberry3:~#

**Comment by yomguy** [comment]
Hi @robnoordt, thanks for the report. 
It seems like `pycurl` is missing.
Did you install all required packages?
```
sudo apt-get install python-pip python-dev python-liblo \
                     python-mutagen python-pycurl python-yaml \
                     libshout3-dev librtmp-dev liblo-dev \
                     libcurl4-openssl-dev libmysqlclient-dev
```

**Comment by yomguy** [comment]
fixed in #92
